README.md

```diff
@@ -1,39 +1,60 @@
 # Beaker
 
-Beaker is an acceptance testing harness for Puppet PE and other Puppet Projects.  It can also be used as a virtual machine provisioner - setting up machines, running any configuration on those machines and then exiting.
+Beaker is a test harness focused on acceptance testing via interactions
+between multiple (virtual) machines. It provides platform abstraction between
+different Systems Under Test (SUTs), and it can also be used as a virtual machine
+provisioner - setting up machines, running any commands on those machines
+and then exiting.
 
-Beaker runs tests written in Ruby with an additional DSL API.  This gives you access to all standard Ruby along with acceptance testing specific commands.
+Beaker runs tests written in Ruby, and provides additional DSL methods.  This
```

Comment:
Maybe define `DSL`, in case people mistake it for [this](https://en.wikipedia.org/wiki/Digital_subscriber_line)?